src/service.js

```diff
@@ -281,7 +281,9 @@ class Service {
             .update(omit(query, '$populate'), data, options)
             .lean(this.lean)
             .exec()
-            .then(() => this._getOrFind(id, findParams));
+            .then(() => {
```

Comment:
Did this need to be changed? One line arrow functions usually have an implicit return so it should be equivalent (and I prefer the short version).